eth_test_parser/src/trie_builder.rs

```diff
@@ -91,7 +91,6 @@ impl TestBody {
         let contract_code: HashMap<_, _> = self
             .pre
             .into_iter()
-            .filter(|(_, pre)| pre.code.0.is_empty())
```

Comment:
I guess this was inverted, meant to filter out empty code? The zkEVM does currently expect a code input even for empty code though, so I think we should just not filter out anything.

Reply:
Agreed.